tools/topology/topology2/cavs-sdw.conf

```diff
@@ -95,7 +95,7 @@ IncludeByKey.SDW_DMIC {
 #
 #ALH Index: 0, Direction: duplex
 Object.Dai {
-	ALH."2" {
+	ALH.0 {
```

Comment:
I can't follow what the formula is in the commit message. there's a stray '~' character in there. Can we fix?

Reply:
> I can't follow what the formula is in the commit message. there's a stray '~' character in there. Can we fix?

Fixed